src/plugins/controls/public/react_controls/control_group/get_control_group_factory.tsx

```diff
@@ -184,14 +188,11 @@ export const getControlGroupEmbeddableFactory = (services: {
             onSave: ({ type: controlType, state: initialState }) => {
               controlsManager.api.addNewPanel({
                 panelType: controlType,
-                initialState: options?.controlInputTransform
-                  ? options.controlInputTransform(
-                      initialState as Partial<ControlGroupSerializedState>,
-                      controlType
-                    )
+                initialState: settings?.controlInputTransform
```

Comment:
Your original changes renamed `controlInputTransform` to `controlStateTransform`? Should that be keep? `controlStateTransform` is still used in an example.

Reply:
Yup, fixed in a follow up - just got missed when fixing the merge conflicts that https://github.com/elastic/kibana/pull/190273 caused